docs/dev/api_integration.md

```diff
@@ -5,161 +5,52 @@ type: documentation
-| Name               | The video file name must be `partnerName-appName.mp4`.                                |
-| Video Length       | The maximum video length is 60 seconds.                                               |
-| Description        | The maximum number of characters allowed is 300.                                      |
-
-Technology Partners can add up to eight images (seven if you are including a video) in an integration tile's media carousel.
-
-The images must meet the following requirements:
-
-| Image Requirements | Description                                                                                                                                       |
-|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
-| Type               | `.jpg` or `.png`.                                                                                                                                 |
-| Size               | The average is around 500KB. The maximum image size is 1MB.                                                                                       |
-| Dimensions         | The aspect ratio must be 16:9 exactly and fit these specifications:<br><br>- Width: 1440px<br>- Minimum height: 810px<br>- Maximum height: 2560px |
-| Name               | Use letters, numbers, underscores, and hyphens. Do not use spaces.                                                                           |
-| Color Mode         | RGB                                                                                                                                               |
-| Color Profile      | sRGB                                                                                                                                              |
-| Description        | The maximum number of characters allowed is 300.                                                                                                  |
-
-Follow this template to define the `media` object in the media carousel which includes an image, a video thumbnail, and a video:
-
-{{< code-block lang="json" filename="manifest.json" collapsible="true" >}}
-"media": [
-      {
-        "media_type": "image",
-        "caption": "A Datadog API Integration OOTB Dashboard",
-        "image_url": "images/integration_name_image_name.png"
-      },
-      {
-        "media_type": "video",
-        "caption": "A Datadog API Integration Overview Video",
-        "image_url": "images/integration_name_video_thumbnail.png",
-        "vimeo_id": 123456789
-      },
-    ],
-{{< /code-block >}}
+## Setup
 
-For more information, see [Integrations Assets Reference][20].
+### Prerequisites
 
-## Open a pull request
+- You must implement OAuth in order to submit data or pull data out of Datadog. 
+- You must support all [Datadog sites][12].
```

Comment:
Not sure if this is a prerequisite or more so part of the development process when implementing OAuth.

Reply:
agree - removing this section entirely and mentioning sites in the oauth section